DQMOffline/EGamma/python/egammaDQMOffline_cff.py

```diff
@@ -31,9 +31,22 @@
 zmumugammaOldAnalysis.analyzerName = cms.string('zmumugammaOldValidation')
 zmumugammaOldAnalysis.phoProducer = cms.InputTag('photons')
 
-
-
+# HGCal customizations
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
+stdPhotonAnalysisHGCal = stdPhotonAnalysis.clone()
+stdPhotonAnalysisHGCal.ComponentName = 'stdPhotonAnalyzerHGCalFromMultiCl'
+stdPhotonAnalysisHGCal.analyzerName = 'stdPhotonAnalyzerHGCalFromMultiCl'
+stdPhotonAnalysisHGCal.phoProducer = 'photonsFromMultiCl'
+stdPhotonAnalysisHGCal.isolationStrength = 2
+stdPhotonAnalysisHGCal.etaMin = -3.0
+stdPhotonAnalysisHGCal.etaMax = 3.0
+stdPhotonAnalysisHGCal.maxPhoEta = 3.0
 
 egammaDQMOffline = cms.Sequence(photonAnalysis*stdPhotonAnalysis*zmumugammaOldAnalysis*zmumugammaAnalysis*piZeroAnalysis*electronAnalyzerSequence)
+egammaDQMOfflineHGCal = egammaDQMOffline.copy()
+egammaDQMOfflineHGCal += cms.Sequence(stdPhotonAnalysisHGCal)
```

Comment:
Not sure I understand what you mean: these modules consumes products that are produced only when the HGCal era is used. Please clarify